examples/top_block.py

```diff
@@ -6,9 +6,9 @@
 #
 # GNU Radio Python Flow Graph
 # Title: Top Block
-# GNU Radio version: v3.10-compat-xxx-xunknown
+# GNU Radio version: 3.10.1.1
 
-from distutils.version import StrictVersion
+from packaging.version import Version as StrictVersion
```

Comment:
I would rather see this renamed in the file, `StrictVersion` for those that know python is very old and at this point going to be deprecated as with the rest of the `distutils` package in upcoming versions of python.  Aliasing like this makes me think that `StrictVersion` is still being used throughout the code.

Reply:
top_block.py is generated code from GNURadio.  I have no control over what and how they do that.  So this file cannot really be reviewed.

Reply:
They must have been lazy about this update and just didn't want to update the references... shame on GNURadio :grin: but all good here!